File: solutions/go/line-up/1/line_up.go
package lineup

import "fmt"

func Format(name string, number int) string {
	end := number % 10
    tail := number % 100
    if tail != 11 && tail != 12 && tail != 13 {
        switch end {
            case 1:
            	return fmt.Sprintf("%s, you are the %dst customer we serve today. Thank you!", name, number)
            case 2:
            	return fmt.Sprintf("%s, you are the %dnd customer we serve today. Thank you!", name, number)
            case 3:
            	return fmt.Sprintf("%s, you are the %drd customer we serve today. Thank you!", name, number)
        	default:
            	return fmt.Sprintf("%s, you are the %dth customer we serve today. Thank you!", name, number)
        }
    }
    return fmt.Sprintf("%s, you are the %dth customer we serve today. Thank you!", name, number)
}

File: solutions/python/line-up/1/line_up.py
def line_up(name, number):
    end = number % 10
    tail = number % 100
    if tail != 11 and tail != 12 and tail != 13: 
        match end:
            case 1:
                return f"{name}, you are the {number}st customer we serve today. Thank you!"
            case 2:
                return f"{name}, you are the {number}nd customer we serve today. Thank you!"
            case 3:
                return f"{name}, you are the {number}rd customer we serve today. Thank you!"
            case _:
                return f"{name}, you are the {number}th customer we serve today. Thank you!"

    return f"{name}, you are the {number}th customer we serve today. Thank you!"